File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/ProfileId.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id 
 * @param name The player name
 */
data class ProfileId (

        @get:NotNull 
        @JsonProperty("id") val id: String,

        @get:NotNull 
        @JsonProperty("name") val name: String
) {

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/com/github/asyncmc/mojang/authentication/scala/gatling/model/ProfileId.scala

package com.github.asyncmc.mojang.authentication.scala.gatling.model


case class ProfileId (
    _id: String,
    /* The player name */
    _name: String
)
object ProfileId {
    def toStringBody(var_id: Object, var_name: Object) =
        s"""
        | {
        | "id":$var_id,"name":$var_name
        | }
        """.stripMargin
}
